_layouts/partners/faq.html

```diff
@@ -0,0 +1,129 @@
+---
+layout: base
+---
+<header>
+  {% include skip_nav.html %}
+  {% include banner.html %}
+  {% include partners/header.html %}
+</header>
+<main id="main-content">
+    <section>
+        <div class="container partners-container">
+            <div class="grid-row">
+                <h1 class="partners-title">{{ page.title }}</h1>
+                <div class="tablet:grid-col-9 accordion-container">
+                  <div class="partners-border"></div>
+                  <div class="section-title-row">
+                    <div id="general-section" class="section-title-left">{{ page.general_section_title | markdownify}}</div>
+                    <button
+                      class="section-title-right collapse-expand-btn-general usa-button usa-button--unstyled"
+                      onclick="javascript:updateAllAccordions('general')">
+                      Expand all
+                    </button>
+                  </div>
+                  <div class="usa-accordion usa-accordion--bordered margin-bottom-5">
+                      {% for item in page.general_accordion %}
+                        <h3 class="usa-accordion__heading">
+                          <button
+                            onclick="updateSingleAccordion(this, 'general')"
+                            class="usa-accordion__button login-accordion-btn-general"
+                            aria-expanded="false"
+                            aria-controls={{ item.id }}
+                            >
+                            {{ item.title }}
+                          </button>
+                        </h3>
+                        <div id={{ item.id }} class="usa-accordion__content login-accordion-content-general">
+                        {{ item.content | markdownify}}
+                        </div>
+                      {% endfor %}
+                  </div>
+                  <a href="#main-content">Back to Top</a>
```

Comment:
This link uses native styles, should add design system styles.

```suggestion
                  <a href="#main-content" class="usa-link">Back to Top</a>
```

(Not a blocker for initial merge)